_test/test_sqw_class/test_binary_ops.m

```diff
@@ -49,23 +49,20 @@ function test_SQW_error_if_operand_is_numeric_but_not_double(obj)
             numeric_array = type_func(ones(size(obj.sqw_obj.data.npix)));
             f = @() obj.sqw_obj + numeric_array;
             ff = @() numeric_array + obj.sqw_obj;
-            assertExceptionThrown(f, 'SQW:binary_op_manager');
-            assertExceptionThrown(ff, 'SQW:binary_op_manager');
+            assertExceptionThrown(f, 'SQWDNDBASE:binary_op_manager');
+            assertExceptionThrown(ff, 'SQWDNDBASE:binary_op_manager');
         end
     end
 
-    function test_adding_sqw_and_dnd_objects_1st_operand_is_sqw_returns_sqw(obj)
-        out = obj.sqw_obj + obj.dnd_obj;
-
-        assertTrue(isa(out, 'sqw'));
+    function test_SQW_error_if_both_objects_are_sqw_with_no_pixel_data(obj)
```

Comment:
One extra test in master, one different extra test in new sqw; next commit will reconcile